modin/utils.py

```diff
@@ -472,7 +525,7 @@ def try_cast_to_pandas(obj, squeeze=False):
     return obj
 
 
-def wrap_into_list(*args, skipna=True):
+def wrap_into_list(*args: Any, skipna: bool = True) -> List[Any]:
```

Comment:
Should `*args*` be a `Sequence` as in other methods?

Reply:
mypy wants `*args` and `**kwargs` annotated with the type of the elements, rather than the type of the container (https://mypy.readthedocs.io/en/stable/getting_started.html#more-function-signatures).